src/RenderView.tsx

```diff
@@ -16,10 +16,40 @@ const THREEzAxis = new THREE.Vector3(0, 0, 1);
 
 interface Props {
 	data: MarkerFileData;
+	forceData: ForceFileData;
 	frame: number;
 }
 
 export default function RenderView(props: Props) {
+	// force data code **************************************
+	/* Get an array of two un-positioned force vector meshes */
+	const forceMeshes = useMemo(() => {
+		const mesh1 = new THREE.Mesh(new THREE.BoxGeometry(0.05,0.05,0.05),new THREE.MeshBasicMaterial());
+		const mesh2 = new THREE.Mesh(new THREE.BoxGeometry(0.05,0.05,0.05),new THREE.MeshBasicMaterial());
+		mesh1.visible = false;
+		mesh2.visible = false;
+		return [mesh1,mesh2];
+	 }, []);
```

Comment:
this line (and several others) have an extra space at the start that shouldn't be there